### PR TITLE
added Topaz to integrations

### PR DESCRIPTION
**Pull request by ogazitt**
Signed-off-by: Omri Gazitt <ogazitt@gmail.com>

This PR adds Topaz to the Ecosystem page.

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -1356,6 +1356,30 @@ integrations:
...
+      - topaz
+    code:
+      - https://github.com/aserto-dev/topaz
+    tutorials:
+      - https://www.topaz.sh
+      - https://github.com/aserto-dev/topaz#quickstart
+      - https://www.topaz.sh/docs/intro
+    blogs:
+      - https://www.aserto.com/blog/topaz-oss-cloud-native-authorization-combines-opa-zanzibar
+    videos:
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:1380]: Sorry this doesn't count as a video 😄 That section is meant for talks, or demonstrations.

File: docs/website/data/integrations.yaml [hunk 2]
```diff
@@ -1356,6 +1356,30 @@ integrations:
...
+    inventors:
+      - aserto
+    software:
+      - topaz
+    code:
+      - https://github.com/aserto-dev/topaz
+    tutorials:
+      - https://www.topaz.sh
+      - https://github.com/aserto-dev/topaz#quickstart
+      - https://www.topaz.sh/docs/intro
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:1377]: Let's not repeat it -- Let's add `https://www.topaz.sh` or `https://www.topaz.sh/docs/intro`, not both.

Also the quickstart section on github is identical to https://www.topaz.sh/docs/getting-started, so I don't think the repo needs to be linked again .here.

File: docs/website/data/integrations.yaml [hunk 3]
```diff
@@ -1356,6 +1356,30 @@ integrations:
     videos:
       - https://www.youtube.com/watch?v=RJkgmdjJn_w
 
+  topaz:
+    title: Topaz
+    description:  |
+      Topaz is an open source authorization service providing fine grained, real-time, policy based access control for applications and APIs. 
+      Topaz combines the best of OPA and Zanzibar in a single container that you can host as a sidecar or microservice.
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:1363]: This sounds vague. What's the best of Zanzibar? What's the best of OPA? YMMV

**Review by srenatus — COMMENTED** [review]
got a few comments inline 👇 🙃

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (ogazitt) [reply docs/website/data/integrations.yaml:1380]: thanks for the feedback - removed the video

File: docs/website/data/integrations.yaml (hunk 2 above)
Comment (ogazitt) [reply docs/website/data/integrations.yaml:1377]: removed the intro docs, left the microsite and the quickstart (which are different)

File: docs/website/data/integrations.yaml (hunk 3 above)
Comment (ogazitt) [reply docs/website/data/integrations.yaml:1363]: made this more precise

**Review by anderseknert — APPROVED** [review]
This looks OK to me 👍  The DCO check is failing though. @ogazitt can you please squash the commits into a single one, and add a signoff to the commit (as required for CNCF projects)? Let me know if you need any help with that. Once that's done I'll have it merged.

**Pull request merged**